nupm/install.nu

```diff
@@ -0,0 +1,43 @@
+use std log
+
+def nupm-home [] {
+    $env.NUPM_HOME? | default (
+        $env.XDG_DATA_HOME?
+        | default ($nu.home-path | path join ".local" "share")
+        | path join "nupm"
+    )
+}
+
+def prepare-directory [directory: path] {
+    rm --recursive --force $directory
+    mkdir $directory
+}
+
+def copy-directory-to [destination: path] {
+    let source = $in
+
+    log info "copying directory"
+    log debug $"source: ($source)"
+    log debug $"destination: ($destination)"
+
+    ls --all $source | where name != ".git" | each {|it|
+        log debug ($it.name | str replace $source "" | str trim --left --char "/")
```

Comment:
char path_sep instead of "/"

Reply:
good catch :+1: 
i thought i was in a URL probably :eyes:

Reply:
should be fixed in 38e3ba6